.circleci/config.yml

```diff
@@ -197,6 +197,47 @@ jobs:
       - runtests:
           platform: conscrypt
 
+  docs:
+    machine:
+      image: circleci/classic:latest
+    parallelism: 1
+    steps:
+      - checkout # check out source code to working directory
+      - run: |
+          sudo mkdir -p /usr/local/bin
+          sudo chown -R circleci:circleci /usr/local/bin
+      - run: |
+          sudo mkdir -p /usr/local/lib/python3.6/site-packages
+          sudo chown -R circleci:circleci /usr/local/lib/python3.6/site-packages
+      - restore_cache:
+          key: pipdeps-{{ .Branch }}
+      - run: sudo pip install mkdocs mkdocs-material
+      - save_cache:
+          key: pipdeps-{{ .Branch }}
+          paths:
+            - "/usr/local/bin"
+            - "/usr/local/lib/python3.6/site-packages"
+      - run: |
+          ./gradlew \
+            :mockwebserver:dokka \
+            :okhttp-dnsoverhttps:dokka \
+            :okhttp-logging-interceptor:dokka \
+            :okhttp-sse:dokka \
+            :okhttp-tls:dokka \
+            :okhttp-urlconnection:dokka \
+            :okhttp:dokka
+      - run: |
+          cat README.md | grep -v 'project website' > docs/index.md
```

Comment:
I’m unhappy that this code is now duplicate here and in deploy_website.sh. Can you just invoke that script?